docs/en/index.md

```diff
@@ -16,14 +20,14 @@ Each time you run User Sync, it looks for differences between the
 user and group information in the two systems, and updates the
 Adobe directory to match the information in your directory.
 
-# Setup and Success Guide
+# [Setup and Success Guide](success-guide/index.md)
```

Comment:
It's tempting to have the headers be links, but it really doesn't format well.  I think just let them be headers and keep the links in the text.